docusaurus.config.js

```diff
@@ -22,17 +22,27 @@ const config = {
       /** @type {import('@docusaurus/preset-classic').Options} */
       ({
         docs: {
+          breadcrumbs: false,
+          lastVersion: "current",
           sidebarPath: require.resolve("./sidebars.js"),
           // Please change this to your repo.
           editUrl: "https://github.com/LIT-Protocol/docs/edit/main/website/",
           routeBasePath: "/",
+          versions: {
+            '2.0': {
+              badge: false,
+              label: 'v2.x.x',
```

Comment:
maybe this should be 
```suggestion
              label: '< v2.x.x',
```
to capture versions `0.x.x` and `1.x.x`

Reply:
@hwrdtm the content changes between js-sdk v3 and v2.

Reply:
right - just wondering which version would the users using `1.x.x` know to refer to?

Reply:
we launched the ts sdk with 2.x.x^ and removed lit-js-sdk references. as a lesson learned with the lit-js-sdk deprecation, we should keep old content and introduce doc versioning moving forward